explainers/subresource-loading.md

```diff
@@ -80,53 +79,37 @@ Developers will write
 <link
   rel="webbundle"
   href="https://example.com/dir/subresources.wbn"
-  scope="https://example.com/dir/scope/"
+  resources="https://example.com/dir/a.js https://example.com/dir/b.js https://example.com/dir/c.png"
 />
 ```
 
-to tell the browser that subresources under `https://example.com/dir/scope/` can
-be found within the `https://example.com/dir/subresources.wbn` scope.
+to tell the browser that subresources specified in `resources` attribute can
+be found within the `https://example.com/dir/subresources.wbn` bundle.
 
 When the browser parses such a `link` element, it:
 
-1. Fetches the specified Web Bundle, `/dir/subresources.wbn`.
+1. Fetches the specified Web Bundle, `https://example.com/dir/subresources.wbn`.
 
-2. Records the scope and _delays_ fetching any other subresource within that
-   scope.
+2. Records the `resources` and _delays_ fetching a subresource specified there if either
 
-3. As the bundle arrives, the browser fulfills those other pending subresource
-   fetches from the bundle's contents.
-
-4. If a fetch within the bundle's declared scope isn't actually contained inside
-   the bundle, it's probably better to fail that fetch than to go to the
-   network, since it's easier for developers to fix a deterministic network
-   error than a performance problem.
-
-As mentioned above, several aspects of this design are tentative. [Alternate
-Designs](#alternate-designs) suggests several ways we're considering changing
-it.
+   - a subresource's origin is the [same origin](https://html.spec.whatwg.org/#same-origin)
+     as the bundle's origin, or
+   - a subresource's URL is [urn::uuid](https://tools.ietf.org/html/rfc4122) URL.
```

Comment:
We should say something about whether the the bundle subresources are entered into the HTTP cache. I think it's moderately important that we not put the `urn:uuid` resources into the cache, to prevent it from being a problem if a site re-uses those UUIDs. The origin+path restriction makes it fairly safe to cache the `https:` subresources, but we also don't want to be causing extra IPCs to fetch those from the cache. This interacts with the "Subsequent loading and Caching" section and might belong there.

A note about this would also make a good start for a security considerations section.

Reply:
Thanks! Yes, it is a good idea to have a security consideration section in this explainer.
Let me try to add a security consideration section as another PR.